src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -119,6 +128,15 @@ public String postApproval(@PathVariable String companyNumber,
                 return getTemplateName();
             }
 
+            String uri = TRANSACTIONS_URI.expand(transactionId).toString();
```

Comment:
Should this logic be in transaction service too?

Reply:
Moved the logic into service class